README.md

```diff
@@ -392,6 +392,110 @@ The default port the Pushgateway is listening to is 9091. The path looks like:
 * For example to wipe all metrics from the Pushgateway:
 
         curl -X PUT http://pushgateway.example.org:9091/api/v1/admin/wipe
+
+## Rest API
+
+Rest API provides Rest based interface for accessing metrics and build information. 
+
+### URL
+
+    /api/<API_VERSION>/<HANDLER>
+    
+ * Available endpoints:
+ 
+| HTTP_METHOD| API_VERSION |  HANDLER | DESCRIPTION |
+| :-------: |:-------------:| :-----:| :----- |
+| GET     | v1 | status |  Returns build information in JSON format. |
```

Comment:
"Returns build information, command line flags, and the start time in JSON format."